src/commands/account/create_account/fund_myself_create_account/sign_as/mod.rs

```diff
@@ -0,0 +1,446 @@
+            }
+        }
+    }
+    None
+}
+
+fn optional_signer_account_view(
+    context: &crate::commands::account::create_account::CreateAccountContext,
+    account_id: near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<Option<near_primitives::views::AccountView>> {
+    for network in context.config.networks.iter() {
+        let _is_signer_account_id = loop {
+            match tokio::runtime::Runtime::new().unwrap().block_on(
+                crate::common::get_account_state(
+                    network.1.clone(),
+                    account_id.clone(),
+                    near_primitives::types::Finality::Final.into(),
+                ),
+            ) {
+                Ok(optional_account_view) => {
+                    if optional_account_view.is_some() {
+                        return Ok(optional_account_view);
+                    } else {
+                        return Ok(None);
+                    }
+                }
+                Err(near_jsonrpc_client::errors::JsonRpcError::TransportError(_)) => {
+                    println!("\nAddress information not found: A host or server name was specified, or the network connection <{}> is missing. So now there is no way to check if <{}> exists.", network.1.network_name, account_id);
+
+                    let choose_input = vec![
+                        "Yes, I want to check the account_id again.",
+                        "No, I don't want to check the account_id again.",
+                    ];
+                    let select_choose_input = Select::with_theme(&ColorfulTheme::default())
+                        .with_prompt("Do you want to check the account_id again on this network?")
+                        .items(&choose_input)
+                        .default(0)
+                        .interact_on_opt(&Term::stderr())?;
+                    if matches!(select_choose_input, Some(1)) {
+                        break false;
+                    }
+                }
+                Err(near_jsonrpc_client::errors::JsonRpcError::ServerError(
+                    near_jsonrpc_client::errors::JsonRpcServerError::HandlerError(
+                        near_jsonrpc_primitives::types::query::RpcQueryError::UnknownAccount {
+                            ..
+                        },
+                    ),
+                )) => {
+                    break false;
+                }
+                Err(near_jsonrpc_client::errors::JsonRpcError::ServerError(_)) => {
+                    println!(
+                        "Unable to verify the existence of account <{}> on network <{}>",
+                        account_id, network.1.network_name
+                    );
+                    break false;
+                }
+            }
+        };
+    }
+    Ok(None)
+}
+
+async fn is_signer_account_id(
+    network_config: &crate::config::NetworkConfig,
+    account_id: &near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<bool> {
```

Comment:
```suggestion
async fn validate_signer_account_id(
    network_config: &crate::config::NetworkConfig,
    account_id: &near_primitives::types::AccountId,
) -> color_eyre::eyre::Result<()> {
```

There is no need to return `bool` when it can only return `true`, so we can use `()`

Reply:
done